src/main/scala/intellij/haskell/external/component/CabalConfigComponent.scala

```diff
@@ -88,14 +105,18 @@ object CabalConfigComponent {
     }
   }
 
-  private def getResolverFromLocalPkgdb(project: Project): Option[String] = {
-    StackCommandLine.runCommand(Seq("path", "--local-pkg-db"), project).flatMap(output => {
-      output.getStdout match {
-        case LocalLtsPkgdbResolverPattern(resolver) => Some(resolver)
-        case LocalNightlyPkgdbResolverPattern(resolver) => Some(resolver)
-        case _ => None
-      }
-    })
+  private def getResolverFromStackYamlFile(project: Project): Option[String] = {
+    try {
+      Option(new Yaml()
+        .load(new FileInputStream(new File(getYamlFilePath(project))))
+        .asInstanceOf[java.util.Map[String, Any]]
+        .get("resolver"))
+        .map(_.asInstanceOf[String])
```

Comment:
Yes, get can return null, that's why I wrap it up with a `Option`, I will also parse default cabal config file if can not get `resolver` from Stack YAML file 😄